Application/EdFi.Ods.AdminApi/Features/OdsInstanceContext/AddOdsInstanceContext.cs

```diff
@@ -0,0 +1,74 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+using AutoMapper;
+using Swashbuckle.AspNetCore.Annotations;
+using EdFi.Ods.AdminApi.Infrastructure;
+using EdFi.Ods.AdminApi.Infrastructure.Database.Commands;
+using EdFi.Ods.AdminApi.Infrastructure.Database.Queries;
+using FluentValidation;
+
+
+namespace EdFi.Ods.AdminApi.Features.OdsInstanceContext;
+
+public class AddOdsInstanceContext : IFeature
+{
+    public void MapEndpoints(IEndpointRouteBuilder endpoints)
+    {
+        AdminApiEndpointBuilder
+           .MapPost(endpoints, "/odsInstanceContexts", Handle)
+           .WithDefaultDescription()
+           .WithRouteOptions(b => b.WithResponseCode(201))
+           .BuildForVersions(AdminApiVersions.V2);
+    }
+
+    public async Task<IResult> Handle(Validator validator, IAddOdsInstanceContextCommand addOdsInstanceContextCommand, IMapper mapper, AddOdsInstanceContextRequest request)
+    {
+        await validator.GuardAsync(request);
+        var addedOdsInstanceContext = addOdsInstanceContextCommand.Execute(request);
+        return Results.Created($"/odsInstanceContexts/{addedOdsInstanceContext.OdsInstanceContextId}", null);
+    }
+
+
+    [SwaggerSchema(Title = "AddOdsInstanceContextRequest")]
+    public class AddOdsInstanceContextRequest : IAddOdsInstanceContextModel
+    {
+        [SwaggerSchema(Description = FeatureConstants.OdsInstanceContextOdsInstanceIdDescription, Nullable = false)]
+        public int OdsInstanceId { get; set; }
+        [SwaggerSchema(Description = FeatureConstants.OdsInstanceContextContextKeyDescription, Nullable = false)]
+        public string? ContextKey { get; set; }
+        [SwaggerSchema(Description = FeatureConstants.OdsInstanceContextContextValueDescription, Nullable = false)]
+        public string? ContextValue { get; set; }
+    }
+
+    public class Validator : AbstractValidator<AddOdsInstanceContextRequest>
+    {
+        private readonly IGetOdsInstanceQuery _getOdsInstanceQuery;
+        
+        public Validator(IGetOdsInstanceQuery getOdsInstanceQuery)
+        {
+            _getOdsInstanceQuery = getOdsInstanceQuery;
+            
```

Comment:
@lchavarriaga 
We have to validate the unique constrain on combination of "OdsInstanceId", "ContextKey" and "ContextValue".  {
  "odsInstanceId": 1,
  "contextKey": "SchoolYear",
  "contextValue": "2009"
}